ccdl.py

```diff
@@ -332,10 +332,17 @@ def runccdl():
         print('Adobe HyperDrive installer not found.\nPlease make sure the Creative Cloud app is installed.')
         exit(1)
 
-    if platform.machine() == 'arm64':
-        ism1 = questiony('Do you want to make M1 native packages')
+    if args.arch:
+        if args.arch == "arm64" or args.arch == "m1" or args.arch == "as":
```

Comment:
change `if args.arch ==` to `if args.arch.lower() ==` so Capitalization don't matter.